common/src/main/java/feast/common/logging/AuditLogger.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.common.logging;
+
+import feast.common.config.LoggingProperties.AuditLogProperties;
+import feast.common.logging.AuditLogEntry.FeastInstance;
+import java.util.Collection;
+import lombok.Getter;
+import lombok.extern.log4j.Log4j2;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.Marker;
+import org.apache.logging.log4j.MarkerManager;
+
+@Log4j2
+@Getter
+public class AuditLogger {
+  /** Defines the output formats that the Audit Logger can produce logs in */
+  public static enum LogFormat {
```

Comment:
Shouldn't the format be a part of the logger configuration xml instead of a part of the code base?

Reply:
The idea was to expose the ability to configure the logger with the most common use cases without interacting with the logging configuration.

Reply:
As mentioned in the response above, I think we can use the application.yml for configuring logging without writing our own logformat configuration.

Reply:
Removed.